### PR TITLE
TW 326 Update interceptor page with cookie sync and cookie capture workflows

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Updates https://learning.postman.com/docs/sending-requests/capturing-request-data/interceptor/ to include new screenshots and workflows for syncing and capturing cookies

### CODE REVIEW

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 1]
```diff
@@ -129,21 +133,45 @@ The **Requests** and **Cookies** tabs display requests, responses, and cookies,
...
+1. In the **Via Interceptor** tab, select **Capture cookies**.
 
-To sync cookies for the domain only, you can add **https://** in front of the domain, such as `https://example.com`.
+    <img src="https://assets.postman.com/postman-docs/interceptor-capture-cookies-checkmark.jpg" alt="Capture cookies" width="300px"/>
 
-Interceptor keeps cookies for a fixed set of domains in sync, updating Postman from the browser when cookies update, including deletions. Due to this, you can use browser sessions to make API calls in Postman. You will not be able to save these to Postman’s history.
+1. When you start a new interceptor session, the captured cookies will be saved along with other network traffic. To view the captured cookies in your Interceptor debug session, select the **Cookies** tab.
 
-You can remove a domain at any time by selecting **X** next to the domain. This will prevent future cookie updates from being synced, but it won’t delete cookies that have already been synced.
+1. To add a captured cookie to your Postman Cookie Jar from an Interceptor debug session, select the cookie then select **+ Add to Cookie Jar**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:172]:
```suggestion
1. To add a captured cookie to your Postman Cookie Jar from an Interceptor debug session, select the cookie and then select **+ Add to Cookie Jar**.
```

**Review by mark-anthony-postman — APPROVED** [review]
One super minor suggestion, otherwise LGTM! ⭐️

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 2]
```diff
@@ -77,16 +77,20 @@ After Interceptor is connected, you can then start an Interceptor debug session,
 
 1. Go to the **Via Interceptor** tab of the **Capture requests and cookies** window.
 
-1. Select **Capture Cookies** if you want to capture cookies in addition to requests during the debug session.
+1. Optionally, Interceptor can sync cookies from specific domains between your browser and Postman. Enter the domain then select **Add domain**. Once you have added the domain or domains, select **Sync cookies**. For more details, see the [Syncing  cookies](#syncing-cookies) section.
```
Comment (claireriley) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:80]:
```suggestion
1. Optionally, Interceptor can sync cookies from specific domains between your browser and Postman. Enter the domain then select **Add domain**. Once you have added the domain or domains, select **Sync cookies**. For more details, see [Syncing cookies](#syncing-cookies).
```

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md [hunk 3]
```diff
@@ -77,16 +77,20 @@ After Interceptor is connected, you can then start an Interceptor debug session,
...
 
-1. Requests will be saved in a debug session in the **History** tab in the sidebar. Select **Save requests to a collection** and select a collection from the list to also save the requests there.
+1. Select **Capture Cookies** if you want to capture cookies in addition to requests during the debug session. You can use these cookies in your Postman requests. For more details, see the [Capturing cookies](#capturing-cookies) section.
```
Comment (claireriley) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:82]: Maybe remove "the" and "section" as above

**Review by claireriley — APPROVED** [review]
Looks great! Noted one thing

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md (hunk 3 above)
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md:82]:
```suggestion
1. Select **Capture Cookies** if you want to capture cookies in addition to requests during the debug session. You can use these cookies in your Postman requests. For more details, see [Capturing cookies](#capturing-cookies).
```

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md (hunk 2 above)
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md]:
```suggestion
1. Optionally, Interceptor can sync cookies from specific domains between your browser and Postman. Enter the domain then select **Add domain**. Once you have added the domain or domains, select **Sync cookies**. For more details, see [Syncing  cookies](#syncing-cookies).
```

File: src/pages/docs/sending-requests/capturing-request-data/interceptor.md (hunk 3 above)
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/capturing-request-data/interceptor.md]:
```suggestion
1. Select **Capture Cookies** if you want to capture cookies in addition to requests during the debug session. You can use these cookies in your Postman requests. For more details, see [Capturing cookies](#capturing-cookies).
```

**Review by shashankawasthi88 — COMMENTED** [review]
Looks good.

**Pull request merged**